homeassistant/components/homematic.py

```diff
@@ -447,6 +450,18 @@ def _system_callback_handler(hass, config, src, *args):
                         ATTR_DISCOVER_DEVICES: found_devices
                     }, config)
 
+    elif src == 'error':
+        _LOGGER.debug("Error: %s", args)
+        (interface_id, errorcode, message) = args
+        try:
+            hass.add_job(hass.bus.async_fire(EVENT_ERROR, {
```

Comment:
Use only `hass.bus.fire`. That fix the problem with value error. We change the handling of `fire` to non blocking too